src/charts/tooltip.js

```diff
@@ -161,9 +161,8 @@ export default function Tooltip (_container, _isLegend = false) {
           }
         ]
 
-        if (_isLegend) {
-          legendData.push({key: "tooltip-label", value: d[keys.LABEL]})
-        }
+        // TO DO: enable this via a config option?
```

Comment:
I think this doesn't belong to this PR

Reply:
correct, it was left over when I branched off #31, will remove it